apps/sup/src/index.ts

```diff
@@ -29,6 +30,23 @@ app.post("/aimodels/sync", ({ error, body }) => {
 
   return true;
 });
+app.get("/islive/:channelName", ({ params, error }) => {
+  const channel = String(params.channelName);
+
+  if (!channel.length) {
+    return error(404);
```

Comment:
If this 404 is the `Not Found`, would be nice to have a static property in a class to keep track of it, so you can be more descriptive, still this is super minor